lib/tests/facts/collection.cc

```diff
@@ -359,3 +360,44 @@ SCENARIO("using the fact collection") {
         }
     }
 }
+
+class collection_override : public collection
+{
+ protected:
+    virtual vector<string> get_external_fact_directories() const override
+    {
+        return {LIBFACTER_TESTS_DIRECTORY "/fixtures/facts/external/ordering/foo"};
+    }
+};
+
+SCENARIO("using the fact collection with a default external fact path") {
```

Comment:
These are new tests verifying that we will load the combination of `get_external_fact_directories` and the argument to `add_external_facts`.